tests/utils.go

```diff
@@ -4197,3 +4196,21 @@ func GenerateRandomMac() (net.HardwareAddr, error) {
 	}
 	return net.HardwareAddr(append(prefix, suffix...)), nil
 }
+func GetUrl(urlIndex int) string {
+	var str string
+
+	switch urlIndex {
+	case AlpineHttpUrl:
+		str = fmt.Sprintf("http://cdi-http-import-server.%s/images/alpine.iso", KubeVirtInstallNamespace)
```

Comment:
Maybe instead of having a hard-coded URL we can get the name of the endpoint from the environment (and fallback to some default maybe) and extract the IP of the endpoint once at the beginning of the suite? This way we are not making any assumptions on the form URL itself...